bandit-github/bandit_github/formatter.py

```diff
@@ -45,11 +45,11 @@ def generate_metrics_table(manager):
     confusion_matrix = get_confusion_matrix(manager)
```

Comment:
@realjanpaulus this is not a _confusion_ matrix, since nothing is being confused. Can you rename/refactor?